cpp/test/ome-xml/CMakeLists.txt

```diff
@@ -64,6 +64,10 @@ if(BUILD_TESTS)
   target_link_libraries(constrained-numeric OME::XML)
   target_link_libraries(constrained-numeric ome-test)
 
+  add_executable(orderedmultimap orderedmultimap.cpp)
+  target_link_libraries(orderedmultimap)
```

Comment:
Does this need to be linked to the OME XML library? - target_link_libraries(orderedmultimap OME::XML)